core/src/main/java/hudson/PluginFirstClassLoader.java

```diff
@@ -45,7 +48,21 @@
     extends AntClassLoader
     implements Closeable
 {
-    
+
+    private final Vector pathComponents;
+
+    public PluginFirstClassLoader() {
+        super();
+
+        try {
+            Field $pathComponents = AntClassLoader.class.getDeclaredField("pathComponents");
+            $pathComponents.setAccessible(true);
```

Comment:
Yeah, I'm generally mystified but wanted to go with the least-disruptive-change-possible in case there was some legitimate reason somewhere for it. I'll add a common ancestor.